shift-circuit-breaker.gemspec

```diff
@@ -28,7 +28,7 @@ Gem::Specification.new do |s|
   s.executables   = `git ls-files -- bin/*`.split("\n").map { |f| File.basename(f) }
 
   s.add_runtime_dependency "activesupport", "~> 5.1", ">= 5.1.4"
-  s.add_runtime_dependency "newrelic_rpm", "~> 4.8", ">= 4.8.0.341"
+  s.add_runtime_dependency "newrelic_rpm", "~> 5.3", ">= 5.3.0.346"
```

Comment:
I'd make this a `add_development_dependency`, it's not required for the gem to operate. You'll see that in the newrelic adapter, it checks if NewRelic is defined before executing.